e2e/connect.test.tsx

```diff
@@ -26,13 +26,13 @@ describe('connect', () => {
     expect(toolbar.textContent).toBe('Connect wallet to swap')
   })
 
-  describe('with jsonRpcEndpoint', () => {
+  describe('with jsonRpcUrlMap', () => {
     it('prompts for wallet connection in the Wallet', async () => {
-      component.rerender(<SwapWidget tokenList={tokens} jsonRpcEndpoint={hardhat.url} />)
+      act(() => component.rerender(<SwapWidget tokenList={tokens} jsonRpcUrlMap={{ 1: [hardhat.url] }} />))
```

Comment:
I'm not super familiar w/ these testing utils, but the [docs ](https://testing-library.com/docs/preact-testing-library/api/#act)seem to indicate the `act` wrapper should not be necessary.

Reply:
think it's because the `Web3Provider` causes issues, we have the same errors in interface atm